src/App.jsx

```diff
@@ -14,33 +16,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const sorted = (goods, sortBy, isReverse) => {
```

Comment:
You can move this function to a separate file then import and use it in the component as a helper or util for soring. 
Also, think of naming. The best way to use `getSortedGoods `or something like that.

Reply:
done, thanks))